examples/apps/attributable-map/README.md

```diff
@@ -0,0 +1,34 @@
+# @fluid-example/attributable-map
+
+**Hit Counter** is a simple demonstration that showcases how to use the `mixinAttributor` and the experimental DDS `AttributableMap`. The user can click on the button to increment the count value by one. Upon updating the count value, the corresponding attribution key will also be updated. The attributor then retrieves the attribution information and stores it, allowing all co-authors to automatically view the latest timestamp at which any author pressed the hit button. The attribution information, which consists of the user name and the timestamp of the most recent edit, is displayed at the bottom. Please refer to [Attributor](../../../packages/framework/attributor/README.md) and [AttributableMap](../../../experimental/dds/attributable-map/README.md) for more details
```

Comment:
could you add single new-lines on sentence breaks? It doesn't change markdown formatting, but makes future git diffs more reasonable.